help/destinations/catalog/crm/zendesk.md

```diff
@@ -164,4 +163,19 @@ All [!DNL Adobe Experience Platform] destinations are compliant with data usage
 
 Additional useful information from the [!DNL Zendesk] documentation is below:
 * [Making your first call](https://developer.zendesk.com/documentation/sales-crm/first-call/)
-* [Custom Fields](https://developer.zendesk.com/api-reference/sales-crm/requests/#custom-fields)
\ No newline at end of file
+* [Custom Fields](https://developer.zendesk.com/api-reference/sales-crm/requests/#custom-fields)
+
+### Changelog
+
+This section captures the functionality and significant documentation updates made to this destination connector.
+
++++ View changelog
+
+|Release month|Update type|Description|
```

Comment:
The changelog section needs to provide clearer information for a reader. Try to read the section from a customer point of view, and see if you understand what the actual change was. There are also some typos in this section.

Let's rephrase it to something like this:

|Release month|Update type|Description|
|---|---|---|
|April 2023|Documentation update| <ul><li>We updated the [use-cases](#use-cases) section with a clearer example of when customers would benefit from using this destination.</li><li>We updated the [mapping](#mapping-considerations-example) section to reflect the correct required mappings. The `Attribute: last_name` and `Identity: email` target mappings are mandatory for this destination. If these mappings are missing, any other mappings are ignored and not sent to [!DNL Zendesk].</li> <li>We updated the [mapping](#mapping-considerations-example) section with clear examples of both mandatory and optional mappings.</li></ul> |
|March 2023|Initial release|Initial destination release and documentation publish.|